config/routes.rb

```diff
@@ -14,8 +14,18 @@
       get 'decline_invitation'
     end
   end
+
+  resources :projects do
+    member do
+      get 'toggle_status'
+    end
+  end
+
+  delete 'projects/:id/sign_user_out/:user_id' => 'projects#sign_user_out', as: 'sign_user_out_project'
```

Comment:
das soll auch in so einen projects block oder kann das in den selben wie 'toggle_status" ?